src/rust/ensogl/lib/core/src/display/shape/primitive/style_watch.rs

```diff
@@ -177,6 +177,15 @@ impl StyleWatch {
         self.get(path).color().unwrap_or_else(|| FALLBACK_COLOR)
     }
 
+    /// Queries style sheet color and returns it as a string to be used in CSS properties.
+    /// If the path is not found fallbacks to [`FALLBACK_COLOR`].
+    pub fn get_color_for_css<T:Into<Path>>(&self, path:T) -> String {
+        let color      = self.get_color(path);
+        let color_rgba = color::Rgba::from(color);
+        format!("rgba({},{},{},{})",
+                color_rgba.red*255.0,color_rgba.green*255.0, color_rgba.blue*255.0,color_rgba.alpha)
+    }
```

Comment:
there is functon `Rgba::to_javascript_string` - use it instead :) It should work as `styles.get_color(path).to_javascript_string()`. If it doesn't, please add it to Lcha too instead.